crt_portal/cts_forms/templates/forms/report_data.html

```diff
@@ -53,15 +54,15 @@ <h3>{{ question.hatecrime_title }}</h3>
         <h4>{{ question.hatecrime }}</h4>
         {% if report.id %}
             {% for crime in report.hatecrimes_trafficking.all %}
-                <p>{{ crime }}</p>
+                <p>{% trans crime.hatecrimes_trafficking_option %}</p>
```

Comment:
Shouldn't this be taken care of by lazy loading?

https://github.com/usdoj-crt/crt-portal/blob/develop/crt_portal/cts_forms/model_variables.py#L75-L78

Reply:
Addressed in #437